proto/v2/skaffold.proto

```diff
@@ -192,8 +192,9 @@ message BuildSubtaskEvent {
     string id = 1; // id of the subtask which will be used in SkaffoldLog
     string task_id = 2; // id of the task of skaffold that this event came from
     string artifact = 3; // artifact name
-    string status = 4; // artifact build status oneof: InProgress, Completed, Failed
-    ActionableErr actionableErr = 5; // actionable error message
+    string step = 4; // which step of the build for the artifact oneof: Cache, Build, Push
```

Comment:
Just a note: Since v2 is alpha, it is ok to change the  position of the field. Usually, proto should always be backward compatible and should not change the field number.